<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>321</width>
     <height>391</height>
    </rect>
   </property>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>44</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>321</width>
      <height>51</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>321</width>
      <height>51</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>进制转换</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>321</width>
      <height>51</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>321</width>
      <height>51</height>
     </size>
    </property>
    <property name="text">
     <string>10进制四则运算</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>321</width>
      <height>51</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>321</width>
      <height>51</height>
     </size>
    </property>
    <property name="text">
     <string>2/8/10/16进制运算及比较及公倍数公约数</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>250</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>321</width>
      <height>51</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>321</width>
      <height>51</height>
     </size>
    </property>
    <property name="text">
     <string>十进制对数，幂运算，三角函数运算</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>320</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>321</width>
      <height>51</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>321</width>
      <height>51</height>
     </size>
    </property>
    <property name="text">
     <string>一元，反比例，指数对数函数运算</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
